java/src/main/java/io/xpring/demo/XrpDemo.java

```diff
@@ -21,23 +24,34 @@ public static void main(String[] args) throws XpringException {
         String recipientAddress = "X7cBcY4bdTTzk3LHmrKAK6GyrirkXfLHGFxzke5zTmYMfw4";
         BigInteger dropsToSend = BigInteger.valueOf(10);
 
+        // Instantiate an XRPClient connected to the XRP Ledger Testnet
         System.out.println("\nUsing rippled node located at: " + grpcAddress + "\n");
-        XRPClient xrpClient = new XRPClient(grpcAddress, true);
+        XRPClient xrpClient = new XRPClient(grpcAddress, XRPLNetwork.TEST);
 
+        // Get account balance
         System.out.println("Retrieving balance for" +  wallet.getAddress() + "..");
         BigInteger balance = xrpClient.getBalance(wallet.getAddress());
 
         System.out.println("Balance was " + balance + " drops!\n");
 
+        // Send XRP
         System.out.println("Sending:");
         System.out.println("- Drops " + dropsToSend );
         System.out.println("- To: " + recipientAddress);
         System.out.println("- From: " + wallet.getAddress() + "\n");
         String hash = xrpClient.send(dropsToSend, recipientAddress, wallet);
 
+        // Check status of the payment
         System.out.println("Hash for transaction:\n" + hash + "\n");
 
         TransactionStatus status = xrpClient.getPaymentStatus( hash);
         System.out.println("Result for transaction is:\n" + status + "\n");
+
+        // Retrieve full payment history for account
+        System.out.println("Payment history for account " + wallet.getAddress() +":\n");
+        List<XRPTransaction> paymentHistory = xrpClient.paymentHistory(wallet.getAddress());
+        for (XRPTransaction transaction : paymentHistory) {
```

Comment:
This demo may be slightly more slick if we limit this to the 5 most recent transactions (and obviously print that we're doing that)